azure-pipelines.yml

```diff
@@ -137,28 +136,51 @@ steps:
   - task: Docker@2
     displayName: Login to temporary pipeline ACR
     inputs:
-      command: login
-      containerRegistry: $(PIPELINE_CONTAINER_REGISTRY)
+      containerRegistry: 'ASO Pipeline ACR'
+      command: 'login'
 
   - task: Docker@2
     displayName: Build and Push Docker Image to temporary ACR for validation
     inputs:
-      command: buildAndPush
-      containerRegistry: $(PIPELINE_CONTAINER_REGISTRY)
-      repository: $(IMAGE_NAME)
-      Dockerfile: "$(System.DefaultWorkingDirectory)/Dockerfile"    
-      buildContext: $(System.DefaultWorkingDirectory)
-      tags: |
-        $(MAJOR_VERSION).$(MINOR_VERSION).$(PATCH_VERSION)
+      containerRegistry: 'ASO Pipeline ACR'
+      repository: '$(IMAGE_NAME)'
+      command: 'buildAndPush'
+      Dockerfile: '$(System.DefaultWorkingDirectory)/Dockerfile'
+      buildContext: '$(System.DefaultWorkingDirectory)'
+      tags: '$(MAJOR_VERSION).$(MINOR_VERSION).$(PATCH_VERSION)'
+
+#  - task: Kubernetes@1
+#    displayName: Deploy to AKS - Fetch kubeconfig
```

Comment:
Do we need this task anymore? Can it be deleted or is it temporarily disabled? Might be helpful to comment why if it's only temporary.